user/src/App.jsx

```diff
@@ -1,7 +1,15 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom'
+import Footer from './components/footer/Footer';
+import Header from './components/header/Header';
```

Comment:
```suggestion
import Footer from './components/Layout/Footer';
import Header from './components/Layout/Header';
```
Có thể để chung 1 folder là Layout nhé. Với lại nên nhớ đặt tên folder in hoa cho các loại Component nhé